frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/field-panels/PaymentsInputPanel.tsx

```diff
@@ -288,11 +289,17 @@ export const PaymentsInputPanel = (): JSX.Element | null => {
     }
   }, [paymentsField, resetData, setData, setToEditingPayment, setToInactive])
 
-  const paymentDisabledMessage = !isEncryptMode
-    ? 'Payments are not available in email mode forms.'
-    : !isStripeConnected
-    ? 'Connect your Stripe account in Settings to save this field.'
-    : ''
+  const paymentDisabledMessage = !isEncryptMode ? (
+    <Text>Payments are not available in email mode forms.</Text>
+  ) : !isStripeConnected ? (
+    <Text>
+      Connect your Stripe account in{' '}
+      <Link as={ReactLink} to={`settings/payments`}>
```

Comment:
Tangential to your PR, but with deep linking we can do this for MyInfo fields too! @staceytan1998 

<img width="593" alt="image" src="https://github.com/opengovsg/FormSG/assets/59867455/776f8370-67f5-4d0e-a0ef-9ffe47990b84">

Reply:
Yea @staceytan1998 mentioned directing users to payments input panel directly too haha future work bah

Reply:
Aite okay! But for this myinfo one right its just from create drawer to `settings/singpass` which should be doable already right? 😅 just add the link to settings or something in the info message, but yeah we can update all our linkings in another PR!

Reply:
Ohhh you mean from infobox to settings! Yea okays yup it'll enable that haha